compiler/rustc_lint/src/types.rs

```diff
@@ -525,8 +525,12 @@ fn lint_literal<'tcx>(
         ty::Float(t) => {
             let is_infinite = match lit.node {
                 ast::LitKind::Float(v, _) => match t {
+                    // v.as_str().parse().map(f16::is_infinite)
+                    ty::FloatTy::F16 => todo!("f16::is_infinite"),
```

Comment:
this should just be `Ok(false)` with a fixme

Reply:
Thanks, should be all set